internal/cli/runner_install.go

```diff
@@ -0,0 +1,269 @@
+	id                    string   `hcl:"id,optional"`
+	runnerProfileOdrImage string   `hcl:"id,optional"`
+	serverTls             bool     `hcl:"server_tls,optional"`
+	serverTlsSkipVerify   bool     `hcl:"server_tls_skip_verify,optional"`
+	serverRequireAuth     bool     `hcl:"server_require_auth,optional"`
+}
+
+func (c *RunnerInstallCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictNothing
+}
+
+func (c *RunnerInstallCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *RunnerInstallCommand) Flags() *flag.Sets {
+	return c.flagSet(0, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+
+		var runnerPlatforms []string
+		for platformName := range runnerinstall.Platforms {
+			runnerPlatforms = append(runnerPlatforms, platformName)
+		}
+
+		f.EnumVar(&flag.EnumVar{
+			Name:   "platform",
+			Usage:  "Platform to install the Waypoint runner into.",
+			Values: runnerPlatforms,
+			Target: &c.platform,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "server-addr",
+			Usage:  "Address of the Waypoint server.",
+			EnvVar: "WAYPOINT_ADDR",
+			Target: &c.serverUrl,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "odr-image",
+			Usage:   "Docker image for the on-demand runners. If not specified, it defaults to 'hashicorp/waypoint-odr:latest'.",
+			Default: "hashicorp/waypoint-odr:latest",
+			Target:  &c.runnerProfileOdrImage,
+		})
+
+		f.BoolVar(&flag.BoolVar{
+			Name:    "server-tls",
+			Target:  &c.serverTls,
+			Usage:   "If true, the Waypoint runner will connect to the server over TLS.",
+			Default: true,
+		})
+
+		f.BoolVar(&flag.BoolVar{
+			Name:   "server-tls-skip-verify",
+			Target: &c.serverTlsSkipVerify,
+			Usage:  "If true, will not validate TLS cert presented by the server.",
+		})
+
+		f.BoolVar(&flag.BoolVar{
+			Name:   "server-require-auth",
+			Target: &c.serverRequireAuth,
+			Usage:  "If true, will send authentication details.",
+		})
+
+		f.BoolVar(&flag.BoolVar{
+			Name:    "adopt",
+			Usage:   "Adopt the runner after it is installed.",
+			Default: true,
+			Target:  &c.adopt,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "id",
+			Usage:  "If this is set, the runner will use the specified id.",
+			Target: &c.id,
+		})
+
+		sort.Strings(runnerPlatforms)
+
+		for _, name := range runnerPlatforms {
+			platform := runnerinstall.Platforms[name]
+			platformSet := set.NewSet(name + " Options")
+			platform.InstallFlags(platformSet)
+		}
+	})
+}
+
+func (c *RunnerInstallCommand) Help() string {
+	return formatHelp(`
+Usage: waypoint runner install [options]
+
+  Install a Waypoint runner to an existing platform. The platform should be
+  specified as kubernetes, nomad, ecs, or docker.
+
+  With the '-adopt' flag set to true (the default), this command will adopt the
+  runner after it is installed. The install will attempt to install a runner for
+  the server configured in the current Waypoint context.
```

Comment:
Sorry, wording nit.

```suggestion
  Install a Waypoint runner to the specified platform: kubernetes, nomad, ecs, or docker.

  This command will attempt to install a runner for the server configured in the current Waypoint context. It will adopt the runner after installation, unless the `-adopt` flag is set to false.
```